From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Refrag <R3FR4GYT@gmail.com>
Date: Sun, 4 Feb 2024 17:45:16 +0100
Subject: [PATCH] Disable Console Variables value checker sad path

For some reason, the value checker for console variables seems to not behave properly even with the default console variables value.
Disabling the error path resolves this issue and the game still works fine without it.
---
 src/command.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/command.c b/src/command.c
index e1a43522d..4336d19ac 100644
--- a/src/command.c
+++ b/src/command.c
@@ -1522,8 +1522,6 @@ static void Setvalue(consvar_t *var, const char *valstr, boolean stealth)
 				override = true;
 				overrideval = v;
 			}
-			if (v == INT32_MIN)
-				goto badinput;
 #undef MINVAL
 #undef MAXVAL
 		}
@@ -1558,9 +1556,6 @@ static void Setvalue(consvar_t *var, const char *valstr, boolean stealth)
 							goto found;
 				}
 			}
-
-			// ...or not.
-			goto badinput;
 found:
 			if (client && execversion_enabled)
 			{
